components/image-controller/base/kustomization.yaml

```
@@ -0,0 +1,4 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - https://github.com/konflux-ci/konflux-ci/konflux-ci/image-controller/core?ref=df1ab193b2b9f6224a4c09a5a2ee1bc0f3a4ae57
```

```suggestion
  - https://github.com/konflux-ci/konflux-ci/konflux-ci/image-controller?ref=39f5a17d96720095fe0c006b061a3ed87e2652b2
```
Refer to the root kustomization file which already contains all the resources